src/oidc_bridge.rs

```diff
@@ -78,14 +75,16 @@ pub fn request(app: &Config, email_addr: EmailAddress, client_id: &str, nonce: &
         "?",
         "client_id=",
         &utf8_percent_encode(&provider.client_id, QUERY_ENCODE_SET).to_string(),
-        "&response_type=code",
+        "&response_type=id_token",
         "&scope=",
         &utf8_percent_encode("openid email", QUERY_ENCODE_SET).to_string(),
         "&redirect_uri=",
         &utf8_percent_encode(&format!("{}/callback", &app.public_url),
                              QUERY_ENCODE_SET).to_string(),
         "&state=",
         &utf8_percent_encode(&session, QUERY_ENCODE_SET).to_string(),
+        "&nonce=",
+        &utf8_percent_encode(&nonce, QUERY_ENCODE_SET).to_string(),
```

Comment:
Great catch re: the nonce; we should be generating our own for the Broker <-> Google side of the transaction.